catalog/app/containers/Bucket/Overview.js

```diff
@@ -757,89 +757,82 @@ function Head({ req, s3, overviewUrl, bucket, description }) {
   const classes = useHeadStyles()
   const isRODA = !!overviewUrl && overviewUrl.includes(`/${RODA_BUCKET}/`)
   const colorPool = useConst(() => mkKeyedPool(COLOR_MAP))
+  const statsData = useData(requests.bucketStats, { req, s3, bucket, overviewUrl })
+  const pkgStatsData = useData(requests.bucketPkgStats, { req, bucket })
   return (
-    <Data fetch={requests.bucketStats} params={{ req, s3, bucket, overviewUrl }}>
```

Comment:
There is only indentation change after removing top level `<Data />`, right?

Reply:
basically yes, but also the data has been split into two requests